### PR TITLE
Fixes #615 Add ddi table

### LINKED ISSUE #615: DDI Ratio plot: table is missing in the report
Based on https://github.com/pchelle/Test-of-Qualification-DDI-UGT/blob/main/report.md
DDI Ratio output in the report shows Plots, GMFE and measures. But the Table is missing

### PR DESCRIPTION
**Pull request by abdullahhamadeh**

### CODE REVIEW

**Comment by abdullahhamadeh** [comment]
[current status of report](https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/files/7650637/Report.zip)

File: R/qualification-ddi.R [hunk 1]
```diff
@@ -48,12 +49,18 @@ getQualificationDDIPlotData <- function(configurationPlan) {
...
           ratioList[[pkParameter]]$perpetrator <- observedDataFrame[["Perpetrator"]][observedDataFrame$ID %in% observedDataRecordId]
+          ratioList[[pkParameter]]$routePerpetrator <- observedDataFrame[["Route Perpetrator"]][observedDataFrame$ID %in% observedDataRecordId]
+          ratioList[[pkParameter]]$routeVictim <- observedDataFrame[["Route Victim"]][observedDataFrame$ID %in% observedDataRecordId]
           ratioList[[pkParameter]]$victim <- observedDataFrame[["Victim"]][observedDataFrame$ID %in% observedDataRecordId]
+          ratioList[[pkParameter]]$dose <- observedDataFrame[["Dose"]][observedDataFrame$ID %in% observedDataRecordId]
+          ratioList[[pkParameter]]$doseUnit <- observedDataFrame[["Dose Unit"]][observedDataFrame$ID %in% observedDataRecordId]
+          ratioList[[pkParameter]]$description <- observedDataFrame[["Description"]][observedDataFrame$ID %in% observedDataRecordId]
+          ratioList[[pkParameter]]$observedRatio <- observedDataFrame[[ddiPKRatioColumnName[[pkParameter]]]][observedDataFrame$ID %in% observedDataRecordId]
```
Comment (Yuri05) [inline R/qualification-ddi.R:63]: No idea what `observedDataFrame$ID %in% observedDataRecordId` means, but can't it be assigned to a variable first? 
e.g.
```
XYZ <- observedDataFrame$ID %in% observedDataRecordId
ratioList[[pkParameter]]$id <- observedDataFrame[["ID"]][XYZ]
ratioList[[pkParameter]]$studyId <- observedDataFrame[["Study ID"]][XYZ]
...
```

**Comment by abdullahhamadeh** [comment]
@pchelle

Axes labels aren't appearing as before.  Do they have to be set explicitly after the log tick/label changes in tlf?

image

**Comment by Yuri05** [comment]
>current status of report

@abdullahhamadeh Did you temporarily deactivated all time profile plots - or why are they not displayed in the report (only the chapter captions are visible)?

**Pull request merged**